docs/book/src/topics/multitenancy.md

```diff
@@ -19,14 +19,14 @@ Please note NamespaceList will take precedence over Selector if both are set.
 ## Deprecated Identity Types
 
 <aside class="note warning">
-
 <h1> Warning </h1>
-The capability to set credentials using environment variables has been removed, the required approach is to use `AzureClusterIdentity` as seen in the [identities](identities.md) page.
+The deprecated identity types below will be removed in CAPZ release v1.15.0. Please migrate to a supported identity type, as described in the <a href="identities.html">identities</a> documentation.
```

Comment:
I think we need to update the docs to clarify that SP types no longer use AAD pod identity (and remove the docs on AAD Pod Identity altogether).